### PR TITLE
script to check if teledyne and gumstix are connected

### PR DESCRIPTION
**Pull request by marcgabe15**
is the last line right to check for filesystem usage?

### CODE REVIEW

**Comment by RustyBamboo** [comment]
Rename this to like 999-ping-check so that it appears last

File: scripts/100-ping-check [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+ping_checker() {
+	HOST="$1"
+	HOST_PING="$(ping -c 2 $HOST 2>&1 | grep '% packet' | awk -F'[%]' '{print $1}' | awk -F'[ ]' '{print $NF}')"
```
Comment (RustyBamboo) [inline scripts/100-ping-check]: when the host is not available, it takes this over 5 seconds to run, which is a little inconvenient. Any way to reduce this time?

File: scripts/100-ping-check [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	if [ ! -z "$HOST_PING" ]; then
+		if [ $HOST_PING -lt 25 ]; then
+			echo "$HOST......OKAY"
+		else
+			echo "$HOST......NOT OKAY"
+		fi
+	fi
+}
+ping_checker mil-sub-sub8.ad.mil.ufl.edu
```
Comment (RustyBamboo) [inline scripts/100-ping-check]: shouldn't this be gumstix IP?

File: scripts/100-ping-check [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ping_checker mil-com-teledyne-p900.ad.mil.ufl.edu
+btrfs filesystem usage
```
Comment (RustyBamboo) [inline scripts/100-ping-check]: `btrfs filesystem usage /`

**Comment by RustyBamboo** [comment]
Yeah, if the host is not available it takes way to long for the motd to appear

**Comment by RustyBamboo** [comment]
image
But it works

**Comment by marcgabe15** [comment]
ok i added a two second timeout does this work better

**Comment by RustyBamboo** [comment]
Address the above requests, and I think it would convenient to have OKAY and NOT OKAY be in color

File: scripts/100-ping-check (hunk 1 above)
Comment (marcgabe15) [reply scripts/100-ping-check]: i added a timeout of two seconds is that okay?

File: scripts/100-ping-check (hunk 2 above)
Comment (marcgabe15) [reply scripts/100-ping-check]: lol my bad

File: scripts/100-ping-check (hunk 3 above)
Comment (marcgabe15) [reply scripts/100-ping-check]: ill fix that too

File: scripts/100-ping-check (hunk 1 above)
Comment (RustyBamboo) [reply scripts/100-ping-check]: :+1:

**Comment by RustyBamboo** [comment]
Okay cool, looks like it works although still not as fast as I'd like it to be.
Last problem is to reformat the commit messages to match our standard of `TOPIC: present tense description`, which can be done by doing interactive rebase `git rebase -i HEAD~2` and then editing the message

File: scripts/999-ping-check [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+		else
+			echo -e "$HOST......${RED}NOT OKAY${NC}"
+		fi
+	else 
+		echo -e "$HOST...... ${RED} NOT FOUND${NC}"
+	fi
+}
+ping_checker mil-sub-gumstix.ad.mil.ufl.edu
+ping_checker mil-com-teledyne-p900.ad.mil.ufl.edu
+btrfs filesystem usage /
```
Comment (RustyBamboo) [inline scripts/999-ping-check]: `btrfs fi show` seems to be a better command to run as it's more straight to the point of file system usage

**Review by RustyBamboo — APPROVED** [review]

**Pull request merged**